react/week4/github-userSearch/src/components/UserData.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from "react";
+import UserSearch from "./Form";
+import UserList from "./UserList";
+
+function UserData() {
+	const url = `https://api.github.com/search/users?`;

```

Comment:
Variables like this one, is common to name them in upperCase
```suggestion
	const URL = `https://api.github.com/search/users?`;

```